torax/_src/geometry/pydantic_model.py

```diff
@@ -318,6 +324,11 @@ def _validate_model(self) -> typing_extensions.Self:
           '`imas_uri` or `imas_filepath` to be a valid input. Inputs provided: '
           f'{specified_inputs}.'
       )
+    if self.slice_time is not None and self.slice_index != 0:
```

Comment:
why can both of these not be provided? The docstring says that time overrides index so I would assume that a user could provide both.